modules/users/manifests/roryhurley.pp

```diff
@@ -1,6 +1,7 @@
 # Creates the roryhurley user
 class users::roryhurley {
   govuk_user { 'roryhurley':
+    ensure => absent,
```

Comment:
Puppet is complaining about the indentation here

Reply:
Good shout, thanks, I see it now. Have corrected.